todo/settings.py

```diff
@@ -1,28 +1,29 @@
 # If a documented django-todo option is NOT configured in settings, use these values.
 from django.conf import settings
 
-hash = {
+defaults = {
```

Comment:
OK with renaming this hash, but I'm not clear why you renamed it from defaults to settings. The purpose here is to establish a value if the dev didn't set one in settings. This seems to be making it into something other than what it's intended for.

Reply:
Mainly because elsewhere in the code you were asking for `defaults('setting_name')` to get the value of a setting. You're not getting the default at all. You're feting the value of the setting, unless it is not set then returning a default. This is in fact exactly what the rest of Django does, settings all have default values. But to get the value of the setting `setting('setting_name')` is much clearer - it says what you are fetching.

Reply:
Hmm, I do see your point. On the other hand, we have a convention that `settings.py` is where you put your settings, and that is not what this file is for! By giving it the same filename, it implies that it's for storing settings.

Reply:
That is a minor concern to me, as `settings.py` for conf settings is generally very well identifies. `defaults.py` I find wholly inappropriate though. Perhaps moving that small utility into a utility file gets around the whole naming issue. it's a tiny thing. In fact it's not entirely out of place in `settings.py` itself IMHO.